variant-remapping-automation/genome_target_tracker.py

```diff
@@ -0,0 +1,96 @@
+from argparse import ArgumentParser
+
+import requests
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_ensembl
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+remapping_genome_target_table = 'evapro.remapping_genome_target_tracker'
+ensembl_url = 'http://rest.ensembl.org/info/assembly'
+
+
+def get_all_taxonomies_from_eva(private_config_xml_file):
+    taxonomy_list = []
+    with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
+        query = 'SELECT DISTINCT taxonomy_id FROM evapro.taxonomy'
+        for taxonomy in get_all_results_for_query(pg_conn, query):
+            taxonomy_list.append(taxonomy[0])
+
+    return taxonomy_list
+
+
+def get_tax_asm_from_eva(private_config_xml_file):
+    eva_tax_asm = {}
+    with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
+        query = f"""SELECT DISTINCT taxonomy_id, assembly_id FROM {remapping_genome_target_table} 
+        WHERE current_previous='current'"""
+        for tax_id, asm in get_all_results_for_query(pg_conn, query):
+            eva_tax_asm[tax_id] = asm
+
+    return eva_tax_asm
+
+
+def get_tax_asm_from_ensembl(taxonomy_list):
```

Comment:
I'm thinking that it might make more sense to have this function deal with one taxonomy at a time. 
There could be in the future multiple sources of target assembly if we incorporate new communities
We would iterate over them like this 
```python
assembly = get_supported_asm_from_ensembl(taxonomy)
if not assembly:
    assembly = get_supported_asm_from_second_source(taxonomy)
...
```

Reply:
Which actually makes me think that we should record the source (Ensembl in this case) of our target assembly in the database table.

Reply:
changed the method and updated source to tracking table